config/defaults.go

```diff
@@ -17,19 +17,60 @@
 package cfg
 
 import (
-	"github.com/inconshreveable/log15"
+	api "github.com/ChainSafe/gossamer/internal"
+	"github.com/ChainSafe/gossamer/p2p"
+	"github.com/ChainSafe/gossamer/polkadb"
+	"github.com/ChainSafe/gossamer/rpc"
 	"os"
 	"os/user"
 	"path/filepath"
-	"regexp"
 	"runtime"
 )
 
 const (
-	DefaultHTTPHost = "localhost" // Default host interface for the HTTP RPC server
-	DefaultHTTPPort = 8545
+	DefaultHttpHost = "localhost" // Default host interface for the HTTP RPC server
+	DefaultHttpPort = 8545
+
+	// P2P
+	DefaultP2PPort = 7001
+	DefaultP2PRandSeed = int64(33)
+)
+
+var DefaultP2PBootstrap = []string{
+	"/ip4/104.131.131.82/tcp/4001/ipfs/QmaCpDMGvV2BGHeYERUEnRQAwe3N8SzbUtfsmvsqQLuvuJ",
+	"/ip4/104.236.179.241/tcp/4001/ipfs/QmSoLPppuBtQSGwKDZT2M73ULpjvfd3aZ6ha4oFGL1KrGM",
+}
+
+var (
+	// P2P
```

Comment:
These config structs for their respected service (eg config for p2p -> p2p.Config) need to be named differently in order to properly reference them in a `config.toml` file

Reply:
Checkout the toml tags in `config/config.go` (above). I don't see too many restrictions, however I am leaning towards decoupling the config structs from the services to some extent. Lets take this up as we revisit the cli/config handling